test/e2e/antreapolicy_test.go

```diff
@@ -1220,3 +1222,118 @@ func TestAntreaPolicyStatus(t *testing.T) {
+		data["antrea-controller.conf"] = antreaControllerConf
+		antreaAgentConf, _ := data["antrea-agent.conf"]
+		antreaAgentConf = strings.Replace(antreaAgentConf, "#  NetworkPolicyStats: false", "  NetworkPolicyStats: true", 1)
+		data["antrea-agent.conf"] = antreaAgentConf
+	}, true, true); err != nil {
+		t.Fatalf("Failed to enable NetworkPolicyStats feature: %v", err)
+	}
+
+	skipIfAntreaPolicyDisabled(t, data)
+	serverName, serverIPs, cleanupFunc := createAndWaitForPod(t, data, data.createNginxPodOnNode, "test-server-", "")
+	defer cleanupFunc()
+
+	clientName, _, cleanupFunc := createAndWaitForPod(t, data, data.createBusyboxPodOnNode, "test-client-", "")
+	defer cleanupFunc()
+	k8sUtils, err = NewKubernetesUtils(data)
+	failOnError(err, t)
+	p10 := float64(10)
+	intstr80 := intstr.FromInt(80)
+	dropAction := secv1alpha1.RuleActionDrop
+	selectorB := metav1.LabelSelector{MatchLabels: map[string]string{"antrea-e2e": clientName}}
+	selectorC := metav1.LabelSelector{MatchLabels: map[string]string{"antrea-e2e": serverName}}
+	protocol := v1.ProtocolTCP
+
+	var anp = &secv1alpha1.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{Namespace: testNamespace, Name: "np1", Labels: map[string]string{"antrea-e2e": "np1"}},
+		Spec: secv1alpha1.NetworkPolicySpec{
+			AppliedTo: []secv1alpha1.NetworkPolicyPeer{
+				{PodSelector: &selectorC},
+			},
+			Priority: p10,
+			Ingress: []secv1alpha1.Rule{
+				{
+					Ports: []secv1alpha1.NetworkPolicyPort{
+						{
+							Port:     &intstr80,
+							Protocol: &protocol,
+						},
+					},
+					From: []secv1alpha1.NetworkPolicyPeer{
+						{
+							PodSelector: &selectorB,
+						},
+					},
+					Action: &dropAction,
+				},
+			},
+			Egress: []secv1alpha1.Rule{},
+		},
+	}
+
+	if _, err = k8sUtils.CreateOrUpdateANP(anp); err != nil {
+		failOnError(fmt.Errorf("create ANP failed for ANP %s: %v", anp.Name, err), t)
+	}
+
+	sessions := 10
+	var wg sync.WaitGroup
+	for i := 0; i < sessions; i++ {
+		wg.Add(1)
+		go func() {
+			if clusterInfo.podV4NetworkCIDR != "" {
+				cmd := []string{"/bin/sh", "-c", fmt.Sprintf("nc -vz -w 4 %s 80", serverIPs.ipv4.String())}
+				data.runCommandFromPod(testNamespace, clientName, busyboxContainerName, cmd)
+			}
+			if clusterInfo.podV6NetworkCIDR != "" {
+				cmd := []string{"/bin/sh", "-c", fmt.Sprintf("nc -vz -w 4 %s 80", serverIPs.ipv6.String())}
+				data.runCommandFromPod(testNamespace, clientName, busyboxContainerName, cmd)
+			}
+			wg.Done()
+		}()
+	}
+	wg.Wait()
+
+	multiple := 0
+	if clusterInfo.podV4NetworkCIDR != "" {
+		multiple++
+	}
+	if clusterInfo.podV6NetworkCIDR != "" {
+		multiple++
+	}
+
+	time.Sleep(2 * time.Second)
+	totalSessions := sessions * multiple
+	if err := wait.Poll(5*time.Second, defaultTimeout, func() (bool, error) {
+		metric, err := data.crdClient.StatsV1alpha1().AntreaNetworkPolicyStats(testNamespace).Get(context.TODO(), "np1", metav1.GetOptions{})
```

Comment:
out-of-curiosity why the name `metric` and not `stats`?

Reply:
The name was copied from
https://github.com/vmware-tanzu/antrea/blob/b56e8a298fe1bfd903c8f232d3f516546ca2da28/test/e2e/networkpolicy_test.go#L136

It seems from the previous discussion people have made the agreement use `stat` instead of `metric`in the context of `NetworkPolicyStats`. I will fix the  misleading name to both tests

Reply:
The resource and API was first called metrics, I changed to stats which is a more common term for this, but missed some variables in code, you can correct all of them.